package.json

```diff
@@ -3,10 +3,15 @@
   "version": "0.1.0",
   "description": "Endeavors is a values & activities tracking app that lets you:",
   "main": "index.js",
+  "engines": {
+    "node": "^6.11.0",
+    "npm": "^3.10.10"
+  },
```

Comment:
this `engines` property allows us to specify which node/npm we're using. useful because i'm using a different version for a work project and i gotta switch around (using `nvm`, but heard good things about `n` too)